specs/protocol/predeploys.md

```diff
@@ -36,21 +39,26 @@ network forking.
 Predeploy addresses exist in 1 byte namespace `0x42000000000000000000000000000000000000xx`.
 Proxies are set at each possible predeploy address except for the `ProxyAdmin` and the `WETH9`.
 
-| Name                       | Address                                    | Proxied |
-|----------------------------|--------------------------------------------|---------|
-| ProxyAdmin                 | 0x4200000000000000000000000000000000000000 | No      |
-| WETH9                      | 0x4200000000000000000000000000000000000001 | No      |
-| L1Block                    | 0x4200000000000000000000000000000000000002 | Yes     |
-| L2ToL1MessagePasser        | 0x4200000000000000000000000000000000000003 | Yes     |
-| L2CrossDomainMessenger     | 0x4200000000000000000000000000000000000004 | Yes     |
-| GasPriceOracle             | 0x4200000000000000000000000000000000000005 | Yes     |
-| ProtocolVault              | 0x4200000000000000000000000000000000000006 | Yes     |
-| L1FeeVault                 | 0x4200000000000000000000000000000000000007 | Yes     |
-| ValidatorRewardVault       | 0x4200000000000000000000000000000000000008 | Yes     |
-| L2StandardBridge           | 0x4200000000000000000000000000000000000009 | Yes     |
-| L2ERC721Bridge             | 0x420000000000000000000000000000000000000A | Yes     |
-| KromaMintableERC20Factory  | 0x420000000000000000000000000000000000000B | Yes     |
-| KromaMintableERC721Factory | 0x420000000000000000000000000000000000000C | Yes     |
+| Name                       | Address                                    | Introduced | Deprecated | Proxied |
+|----------------------------|--------------------------------------------|------------|------------|---------|
+| ProxyAdmin                 | 0x4200000000000000000000000000000000000000 | Legacy     | No         | No      |
+| WETH9                      | 0x4200000000000000000000000000000000000001 | Legacy     | No         | No      |
+| L1Block (*deprecated*)     | 0x4200000000000000000000000000000000000002 | Bedrock    | Yes        | Yes     |
+| L2ToL1MessagePasser        | 0x4200000000000000000000000000000000000003 | Legacy     | No         | Yes     |
+| L2CrossDomainMessenger     | 0x4200000000000000000000000000000000000004 | Legacy     | No         | Yes     |
+| GasPriceOracle             | 0x4200000000000000000000000000000000000005 | Legacy     | No         | Yes     |
+| ProtocolVault              | 0x4200000000000000000000000000000000000006 | Bedrock    | Yes        | Yes     |
+| L1FeeVault (*deprecated*)  | 0x4200000000000000000000000000000000000007 | Bedrock    | Yes        | Yes     |
+| ValidatorRewardVault       | 0x4200000000000000000000000000000000000008 | Bedrock    | Yes        | Yes     |
```

Comment:
Since "deprecated" is mentioned in the detail description below, it would be better to mark it here as well.

```suggestion
| ProtocolVault (*deprecated*)        | 0x4200000000000000000000000000000000000006 | Bedrock    | Yes        | Yes     |
| L1FeeVault (*deprecated*)           | 0x4200000000000000000000000000000000000007 | Bedrock    | Yes        | Yes     |
| ValidatorRewardVault (*deprecated*) | 0x4200000000000000000000000000000000000008 | Bedrock    | Yes        | Yes     |
```